superset-frontend/package-lock.json

```diff
@@ -21137,8 +21137,7 @@
               "version": "2.1.1",
               "resolved": "",
               "integrity": "sha1-w7M6te42DYbg5ijwRorn7yfWVN8=",
-              "dev": true,
```

Comment:
FWIW, I'm not sure what others do, but I just don't bother committing `package-lock.json` if `package.json` hasn't changed. The thrashing of this file doesn't _usually_ seem to make any notable difference other than some occasional merge conflicts. :)